.Documentation/articles/step-by-step-guides/hello-creator.md

```diff
@@ -0,0 +1,120 @@
+`behaviors` prepare a scene for trainees. `Conditions` are actions expected from the trainee to move to the next step. We expect the trainee to grab the sphere (`condition`), which is visually highlighted (`behavior`), and we expect the trainee to place the sphere (`condition`) at a specific position.
+
+#### Behavior - Highlight the Sphere
+
+> select the ‘Position Sphere’ step in the workflow.
+
+Shift your attention to the opening/already opened ***Step inspector***. You see 3 tabs: ***Behaviors***, ***Transitions***, and ***Unlocked Objects***.
+
+> select the Behavior tab and click the ‘add Behavior’ button.
+> select ‘Highlight Objects’ from the list of Behaviors.
+
+Highlight Object has two properties you can configure: the highlight colour and the Object to highlight.
+
+> drag the sphere object from the hierarchy into the ‘object to highlight’ field.
+
+You get the warning that the 'Sphere is not configured as IHighlightProperty' with a `Fix it` button. The Innoactive Creator simplifies the Training Creation for users not necessarily familiar with unity
+
+> click the `Fix it` button.
+
+The Innoactive Creator will take care of configuring the underlying Unity Objects to make highlighting work.
+
+<img src="../images/step-by-step-guides/behavior.jpg" width="400">
+
+ *Figure 3: The configured Highlight object behavior.*
+
+#### Transition – Grab-and-place the sphere
+
+Placing an object implies you 'grabbed' it before. Thus, we integrated both actions into a single condition called `Snap Object`. It’s called 'snap' because when trainees approach the target position with the object, they can release the object and it will position and rotate itself at the target position.
+
+> select the 'Transitions' tab.
+
+One `step` can have multiple `transitions` to other `steps`. Since your `step` does not have any, it displays by default 'Transition to the End of the Chapter'.
+
+> click ‘Add Condition’ and select ‘Snap Object’ from the list of conditions.
+
+Snap object requires two properties ‘Object to snap’ and ‘Zone to snap into’.
+
+> drag the Sphere object from the hierarchy into the ‘Object to snap’ and click the ‘fix it’ button.
+
+Let’s inspect the `sphere` object: Select the `sphere` object in the ***hierachy*** window and open the ***Inspector*** window (see Fig. 1). You see a list of properties and scripts attached to this object, e.g. 'Box Collider' etc.
+
+![Creator Windows](../images/step-by-step-guides/createSnapZone.jpg "Getting Familiar with Unity - The Creator Layout")
+
+ *Figure 4: Create a snap zone. Select the sphere in the hierachy, click Create Snap Zone button in the unity inspector.*
+
+> scroll down to `Snappable Property`. Click the `Create Snap Zone` button.
+
+A new object `sphere_SnapZone` appeared in the ***hierarchy*** window.
```

Comment:
Did not check in the Unity project, but if the Sphere is upper case then the snapzone will also be capitalized

Reply:
cc @JulieAtInnoactive